naiserator.go

```diff
@@ -215,7 +217,7 @@ func (n *Naiserator) removeOrphanIngresses(logger *log.Entry, app *v1alpha1.Appl
 	return err
 }
 
-func (n *Naiserator) MonitorRollout(app v1alpha1.Application, logger log.Entry, frequency, timeout time.Duration) {
+func (n *Naiserator) MonitorRollout(app *v1alpha1.Application, logger log.Entry, frequency, timeout time.Duration) {
```

Comment:
This should be left as a value instead of a reference. You aren't mutating state here, and shouldn't do that anyway.

Reply:
totally agree! Just a side-effect from some debugging i did.